docs/development/index.rst

```diff
@@ -15,10 +15,13 @@ other open-source repositories.
 
 -  Enhancements to ``seismometer`` undergo thorough scrutiny by maintainers.
 
-Contributors must agree to our contributor licensing agreement, and your
+Contributors must agree to our contributor license agreement, and your
```

Comment:
Can you look into why there wasn't a pages action run for this PR?  
It was intended to run on changes in docs/* so this edit should have triggered it.

Reply:
Added pages.yml to this and expanded capture to more than just the root directories.